makefile

```diff
@@ -21,3 +25,16 @@ clean:
 
 lint:
 	find ./src ./include ./test -type f | xargs python2.7 ./vendor/cpplint/cpplint.py 2>&1 >/dev/null | grep -v 'No copyright message found'
+
+install:
+	$(INSTALL) -D ./build/src/statsdcc $(BINDIR)/statsdcc
+	$(INSTALL) -D ./build/src/proxy $(BINDIR)/statsdcc-proxy
+	$(INSTALL) -D ./etc/proxy.json $(ETCDIR)/proxy.json.example
+	$(INSTALL) -D ./etc/aggregator.json $(ETCDIR)/aggregator.json.example
+
+uninstall:
```

Comment:
I personally like when config files are left unless I explicitly request a full cleanup. This "uninstall" target is not required for packaging, I have just added it for convenience, so it can be modified or removed if necessary.